mono/mini/mini.h

```diff
@@ -2674,11 +2680,16 @@ enum {
 };
 
 const char *mono_arch_xregname (int reg);
+guint32     mono_arch_cpu_enumerate_simd_versions (void);
+
+#ifdef MONO_ARCH_SIMD_INTRINSICS
 void        mono_simd_simplify_indirection (MonoCompile *cfg);
+void        mono_simd_decompose_intrinsic (MonoCompile *cfg, MonoBasicBlock *bb, MonoInst *ins);
```

Comment:
This is really bad naming, mono_simd_decompose_intrinsic and mono_simd_decompose_intrinsics differs by a single letter and do different things.